### PR TITLE
22 : Allow support for dynamically setting version for hive and Spark

### PR DESCRIPTION
**Pull request by maheshk114**

### CODE REVIEW

File: shaded-dependencies/build.sbt [hunk 1]
```diff
@@ -28,9 +28,11 @@ publishArtifact in (Compile, packageSrc) := false
 
 publishArtifact in (Compile, packageBin) := false
 
-val hive_version = "3.1.1"
+val hive_version = sys.props.getOrElse("hive.version", "3.1.1")
```
Comment (somani) [inline shaded-dependencies/build.sbt]: Shall we make the default 3.1.2 instead?

**Review by somani — APPROVED** [review]
+1 from my side, but would prefer @citrusraj to take a look as well.

File: shaded-dependencies/build.sbt (hunk 1 above)
Comment (maheshk114) [reply shaded-dependencies/build.sbt]: done

Comment (citrusraj) [reply shaded-dependencies/build.sbt]: Should the same be done for ORC version as well ?

Comment (maheshk114) [reply shaded-dependencies/build.sbt]: for ORC its already there ..

**Review by citrusraj — APPROVED** [review]

**Pull request merged**